proto/skaffold.proto

```diff
@@ -640,6 +681,10 @@ enum StatusCode {
     DEPLOY_PARSE_MANIFEST_IMAGES_ERR = 1021;
     // Helm config `createNamespace` not available
     DEPLOY_HELM_CREATE_NS_NOT_AVAILABLE = 1022;
+
+    // Test Errors
+
+    // TEST_UNKNOWN_ERROR = 1101;
```

Comment:
Having a commented out proto field seems very strange.

Reply:
Removed the comment.